components/analytics/components/Filter/SelectedFilters.js

```diff
@@ -5,6 +5,7 @@ import { connect } from 'react-redux';
 import get from 'lodash/get';
 import { clearFilterValue } from '../../../../modules/actions';
 import Flex from '../../../shared/Flex';
+import { withErrorToaster } from '../../../../../components/ErrorToaster/ErrorToaster';
```

Comment:
@jyash97 This is anti-pattern, please put `ErrorToaster` component in `batteries` then use it everywhere (make the changes in `arc-dashboard` too). The components will throw error if we use it in some other project.